### ISSUE TITLE
DINO fine-tune hyper-parameters

### ISSUE THREAD

**Issue by srikar2097** [facebookresearch/dino#171]
Hi @mathildecaron31 and @TouvronHugo,
thank you for sharing the code to your inspirational work. I am trying out fine-tuning experiments using DINO (teacher) on following downstream datasets: CIFAR10, CIFAR100, INat18, INat19, Flowers, Cars, ImageNet and am unable to replicate the results. 

I have seen [issue 81](https://github.com/facebookresearch/dino/issues/81) and [issue 144](https://github.com/facebookresearch/dino/issues/144) both of which seem related to CIFAR 10. What about other datasets? I tried the mentioned recipe for CIFAR 10 on INat18 and INat19 and was not able to replicate the results (iNat 18: got 71.02 vs reported 72.6 and iNat19: 76.1 vs 78.6). 

I trained this using latest DINO code and pytorch 1.10.0+cu113. 

Can you please resolve the mystery of downstream datasets fine-tuning hyper-parameters?

**Comment by TouvronHugo** [comment]
Hi @srikar2097,
Thanks for your question. 
It is normal the finetuning configuration for CIFAR is different from the one used for iNaturalist.
The hyper-parameters for iNaturalist are:
--batch-size 128 (for each gpu)
--lr 5e-5 (lr before scaling)
 --epochs 300 
--weight-decay 0.05  
--sched cosine  
 --input-size 224 
--repeated-aug 
--smoothing 0.1 
--warmup-epochs 5 
--aa rand-m9-mstd0.5-inc1 
--mixup .8  
--cutmix 1.0  
--remode pixel 
--reprob 0.25 
 --drop-path 0.1 
--opt adamw 
--warmup-lr 1e-6
This config is for 8 gpus. 
It should be taken into account that the version of the librairy used can have a slight impact on the performance and that there is a more important std on iNat than on ImageNet.
Best,
Hugo

**Comment by srikar2097** [comment]
Hi @TouvronHugo,
much appreciated for sharing the recipe for iNat. Assuming this is used for both INat18, INat19. Two follow up questions: 

1. When you day "there is a more important std on iNat than on ImageNet" you mean multiple runs on val data gives numbers with large variations?
2. What fine-tuning recipe's were used for other datasets? CIFAR100, Flowers, Cars, ImageNet (we covered CIFAR10 and INat18, INat19)

thank you again!
